src/main/scala/lambdaone/collector/Collector.scala

```diff
@@ -0,0 +1,98 @@
+package lambdaone.collector
+
+import cats._
+import cats.data.NonEmptyList
+import cats.implicits._
+import lambdaone.collector.Collector._
+import lambdaone.collector.CollectorEvent.Voted
+import lambdaone.toolbox.{CRUDStore, EventStore, UniqueGen}
+
+trait Collector[F[_], Address, P] {
+
+  def create(ballot: Ballot[P], nickname: String): F[BallotBox[Address, P]]
+
+  def vote(ballotBox: BallotBox[Address, P], vote: Vote[Address, P]): F[ProofOfVote[Address]]
+
+  def validateVote(proofOfVote: ProofOfVote[Address]): F[Boolean]
+
+  def inspect(ballotBox: BallotBox[Address, P]): F[InfluenceAllocation[P]]
+
+}
+
+object Collector {
+
+  type Influence = Double
+
+  type InfluenceAllocation[P] = Map[P, Influence]
+
+  sealed class BinaryProposal
+
+  object BinaryProposal {
+
+    object Yes extends BinaryProposal
+
+    object No extends BinaryProposal
+
+    val AllChoices = NonEmptyList.fromListUnsafe(List(Yes, No))
```

Comment:
```suggestion
    val Ballot = NonEmptyList.fromListUnsafe(List(Yes, No))
```

so to reference it this way: 

```scala
BinaryProposal.Ballot
```

Reply:
`binaryBallot` is defined a few lines further. Will rename it to `ballot`.

Reply:
We need a list of all choices, e.g. to render a voting page.